hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3062,4 +3062,34 @@
       will create intermediate directories.
     </description>
   </property>
+
+  <property>
+    <name>ozone.audit.log.debug.cmd.list.omaudit</name>
+    <value></value>
+    <tag>OM</tag>
+    <description>
+      A comma separated list of OzoneManager commands that are written to the OzoneManager audit logs only if the audit
+      log level is debug.
+    </description>
```

Comment:
I think it would be a good idea to mention some of the possible values that can be included in the comma-separated list.